spot_rl_experiments/experiments/skill_test/episode_configs/pick_32.yaml

```diff
@@ -0,0 +1,69 @@
+# episode21:
```

Comment:
remove this file, is this file for testing? If it is, then please add comments here to know why you want to command out the most of the file.

Reply:
yeah, let's gitignore this file. This file is used for testing in the lab (32-inch tall receptacles).

Reply:
reminder to remove this from the repo and gitignore the `episode_configs` folder